.vscode/settings.json

```diff
@@ -1,6 +1,6 @@
 {
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "[typescript]": {
-    "typescript.preferences.importModuleSpecifier": "relative"
+    "typescript.preferences.importModuleSpecifier": "non-relative"
```

Comment:
Is this change needed? It was added in https://github.com/safe-global/safe-client-gateway/commit/c21e2cf8774803e1aee915ef7a10ca0a5672c974

Reply:
Sorry, a rebase from `main` left. 

Fixed, thanks!